content/ui-ux.md

```diff
@@ -14,11 +14,40 @@ After reading this guide, you'll know:
 5. How to build components that can cope with a variety of different data sources.
 6. How to use animation to keep users informed of changes.
 
-<h2 id="components">UI components</h2>
+<h2 id="view-layers">View layers</h2>
+
+Meteor officially supports three user interface (UI) rendering libraries, [Blaze](blaze.html), [React](react.html) and [Angular](http://www.angular-meteor.com). Blaze was created as part of Meteor when it launched in 2011, React was created by Facebook in 2013, and Angular was created by Google in 2010. All three have been used successfully by large production apps. Blaze is the easiest to learn and has the most full-stack Meteor packages, but React and Angular are more developed and have larger communities.
+
+
+**Syntax**
+- Blaze uses an easy-to-learn [Handlebars](http://handlebarsjs.com)-like template syntax, with logic like `{% raw %}{{#if}}{% endraw %}` and `{% raw %}{{#each}}{% endraw %}` interspersed in your HTML files. Template functions and CSS-selector events maps are written in Javascript files.
+- React uses [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), with which you write your HTML in Javascript. While it doesn't have the logic-view separation most libraries have, it also has the most flexibility. Template functions are TODO
+- Angular uses HTML with [special attribute syntax](https://angular.io/docs/ts/latest/guide/cheatsheet.html) for logic and events. Template functions are TODO
+- React and Angular enforce a better component structure, which makes developing larger apps easier.
```

Comment:
I'd add - although it's possible to extend Blaze to do the same via following conventions, using blaze components/viewmodel, etc.